docs/documentation/server_admin/topics/identity-broker/saml.adoc

```diff
@@ -84,8 +84,16 @@ itself.
 |Validate Signature
 |When *ON*, the realm expects SAML requests and responses from the external IDP to be digitally signed.
 
-|Validating X509 Certificate
-|The public certificate {project_name} uses to validate the signatures of SAML requests and responses from the external IDP.
+|Entity descriptor URL
+|External URL where Identity Provider publishes the `IDPSSODescriptor` metadata. This URL is used to download the Identity Provider certificates when the `Reload keys` or `Import keys` actions are clicked.
+
+|Use entity descriptor URL
+|When *ON*, the certificates to validate signatures are automatically downloaded from the `Entity descriptor URL` and cached in {project_name}. The SAML provider can validate signatures in two different ways. If a specific certificate is requested (usually in `POST` binding) and it is not in the cache, certificates are automatically refreshed from the URL. If all certificates are requested to validate the signature (`REDIRECT` binding) the refresh is only done after a max cache time. The cache can also be manually updated using the action `Reload Keys` in the identity provider page.
```

Comment:
When someone reads this, it is not clear what `max cache time` is and how he can configure it?

IMO it can be nice to introduce some dedicated section in the docs about this mechanism and possibly add some example how the `max cache time` can be used? I suppose the ideal place is this page https://www.keycloak.org/server/all-provider-config . Since new store is removed, it is fine to document the providers related to the old store IMO. When this is documented, then the description of the option `Use entity descriptor URL` on this page `saml.adoc` can contain some link to the provider page to the "infinispan public key loader" provider. WDYT?

Reply:
Yep, I agree. I added the config properties in the infinispan factory and added the link to the documentation. I added also the release notes for 23_0_0 just in case we have time to include it. So done now!